src/test/java/org/isf/patient/test/Tests.java

```diff
@@ -48,6 +48,13 @@
 import org.springframework.data.domain.PageRequest;
 import org.springframework.data.domain.Pageable;
 
+import java.io.File;
+import java.lang.reflect.Field;
+import java.util.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
```

Comment:
I don't believe all the imports added are actually used in the file, are they?  For example, where is **assertThatThrownBy** or **java.io.File**?

Reply:
It seems to me that they are used.